src/board_controller/inc/board.h

```diff
@@ -22,6 +22,7 @@ class Board
 {
 public:
     static std::shared_ptr<spdlog::logger> board_logger;
+    static JNIEnv *java_jnienv;
```

Comment:
I already asked about it but still dont fully understand it. I suppose somewhere file like jni.h should be included in order for it to work. Specially since its not under smth like ifdef android. But I dont see it in headers

Reply:
I added a declaration here to make it clearer:
https://github.com/brainflow-dev/brainflow/blob/eca05e8b714e23d7847bb02ba1b41aedb64f5162/src/board_controller/inc/board_controller.h#L7
I simply gave it the same name as in the jni.h header, so that if the jni library is used, the type is correct.  Because it is just a pointer the type only need be declared, not defined.  It does this explain it?

JNI can be used on any java platform, not just android.  Given this, do you think it would still be good to only enable the code on android?  That would ensure jni.h is available.  Many people find extra #ifdefs to make code harder to read, but many do it as a norm too.

Reply:
yep, now I get it